htdocs/web_portal/GOCDB_monitor/ops_monitor_check.php

```diff
@@ -16,32 +17,24 @@
 foreach ($res as $r){
     $counts[$r["status"]]++;
 }
-/*
-If there is an error, counts["error"] is incremented, e.g.
-Array
-(
-    [ok] => 0
-    [warn] => 0
-    [error] => 1
-)
- */
-
+$ok  = "All GOCDB tests are looking good\n";
+$nok = "GOCDB Web Portal is unable to connect to the GOCDB back end database\n";
```

Comment:
ah this is why we got a slightly weird error message, because as we said - its the only error message it knows how to give.

I'll open an issue for a more generic error message from this page.

Reply:
https://github.com/GOCDB/gocdb/issues/223